account/identity.go

```diff
@@ -12,20 +13,62 @@ import (
 	"golang.org/x/net/context"
 )
 
-// Identity ddenDescribes a unique Person with the ALM
+const (
+	// KeycloakIDP is the name of Keycloak Identity Provider
+	KeycloakIDP string = "kc"
+	// GitHubIDP is the name of GitHub Identity Provider
+	GitHubIDP string = "gh"
+)
+
+// NullUUID can be used with the standard sql package to represent a
+// UUID value that can be NULL in the database
+type NullUUID struct {
```

Comment:
This NullUUID is actually a fixed NullUUID from of github.com/satori/go.uuid. I'm planning to file an issue upstream for that.
And NullXYZ seems to use a name convention for nullable SQL objects in go. See https://golang.org/pkg/database/sql/